bundles/org.openhab.binding.dirigera/README.md

```diff
@@ -0,0 +1,781 @@
+| `id`            | text    | Unique id of this gateway                                  | N/A     | no       |
+| `discovery`     | boolean | Configure if paired devices shall be detected by discovery | true    | no       |
+
+- `ipAddress` - use discovery to obtain this value automatically or enter it manually if known
+- `id` - will be detected automatically after successful pairing
+- `discovery` - will run continuously in the background and detect new, deleted or changed devices. Switch it off to deactivate discovery
+
+### Gateway Pairing
+
+First setup requires pairing the DIRIGERA gateway with openHAB.
+You need physical access to the gateway to finish pairing so ensure you can reach it quickly.
+
+Let's start pairing
+
+1. Add the bridge found in discovery 
+2. Pairing started automatically after creation!
+3. Press the button on the DIRIGERA rear site
+4. Your brdige shall switch to ONLINE 
+
+### Gateway Channels
+
+| Channel         | Type      | Read/Write | Description                                  | Advanced |
+|-----------------|-----------|------------|----------------------------------------------|----------|
+| `pairing`       | Switch    | RW         | Sets DIRIGERA hub into pairing mode          |          |
+| `location`      | Location  | R(W)       | Location in lat.,lon. coordinates            |          |
+| `sunrise`       | DateTime  | R          | Date and time of next sunrise                |          |
+| `sunset`        | DateTime  | R          | Date and time of next sunset                 |          |
+| `statistics`    | String    | R          | Several statistics about gateway activities  |          |
+
+Channel `location` can overwrite GPS position with openHAB location, but it's not possible to delete GPS data.
+See [Gateway Limitations](#gateway-limitations) for further infomration.
+
+### Follow Sun
+
+<img align="right" height="100" src="doc/follow-sun.png">
+
+[Motion Sensors](#motion-sensor) can be active all the time or follow a schedule.
+One schedule is follow the sun which needs to be activated in the DIRIGERA GATEWAY.
+
+## Things
+
+With [DIRIGERA Gateway Bridge](#gateway-bridge) in place things can be connected as mentioned in the [supported things section](#supported-things).
+Things contain generic [configuration](), [properties]() and [channels]() according to their capabilities.
+
+### Generic Thing Configuration
+
+Each thing is identified by a unique id which is mandatory to configure.
+Discovery will automatically identify the id.
+
+| Name              | Type    | Description                         | Default | Required |
+|-------------------|---------|-------------------------------------|---------|----------|
+| `id`              | text    | Unique id of this device / scene    | N/A     | yes      |
+
+### Generic Thing Properties
+
+Each thing has properties attached for identification.
+
+<img align="center" width="500" src="doc/thing-properties.png">
```

Comment:
Don't see much benefit of this picture, if you want to point towards the identification, it is better to point to the specific property field in text.

Reply:
Right, it's _information_, not _identitfication_.
Corrected text and picture